release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -369,6 +369,19 @@ This release adds improvements related to the following components and concepts.
 
 Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
 
+[id="ocp-4-15-egressfirewall-updates-status-management"]
+==== Status management updates for EgressFirewall and AdminPolicyBasedExternalRoute CR
+
+The following updates have been made to the status management of `EgressFirewall` and `AdminPolicyBasedExternalRoute` policy:
+
+** The `status.status` field is set to failure if at least one message reports failure.
+
+** The `status.status` field is empty if no failures are reported and not all nodes have reported their status.
+
+** The `status.status` field is set to success if all nodes report success.
```

Comment:
```suggestion
** The `status.status` field is set to `success` if all nodes report success.
```

Reply:
I did ` ` for both because report doesn't sound like a generic reference more like a literal.